PWGCF/FemtoDream/Core/femtoDreamDetaDphiStar.h

```diff
@@ -34,6 +34,14 @@ namespace femtoDream
 /// \brief Class to check particles for the close pair rejection.
 /// \tparam partOne Type of particle 1 (Track/V0/Cascade/...)
 /// \tparam partTwo Type of particle 2 (Track/V0/Cascade/...)
+
+enum ProngCharmHadron {
+  prong0 = 0,
+  prong1 = 1,
+  prong2 = 2,
+  nprongs = 3
```

Comment:
Enumerator names start with capital letters.

Reply:
Hi @vkucera, I have noted your comments. I will implement these comments in the next version. For now, we would like to see the results on full stats. We already waited too long to approve this code.

Reply:
You have implemented the previous comments in this PR. Why can you not implement these?

Reply:
Hi @vkucera, I put this comment yesterday when I didn't implement anything. I was hoping that previous version will be merge but it was failed. Now, I am working on all the comments

Reply:
OK, thank you.